cmd/tailscale/backend.go

```diff
@@ -48,7 +49,7 @@ type backend struct {
 	lastDNSCfg *dns.OSConfig
 	netMon     *netmon.Monitor
 
-	logIDPublic string
+	logIDPublic atomic.Pointer[logid.PublicID]
```

Comment:
It looks like we only assign to this once on startup, so this doesn't need to be atomic.

Reply:
Done.